editor/plugins/item_list_editor_plugin.h

```diff
@@ -87,6 +89,9 @@ class ItemListPlugin : public Object {
 	virtual void add_item() = 0;
 	virtual int get_item_count() const = 0;
 	virtual void erase(int p_idx) = 0;
+	virtual void undoRedo_add_new_item(int idx)= 0;
+	virtual void undoRedo_re_add_item(unsigned int position, String name, Ref<Texture2D> icon, bool enabled) = 0;
```

Comment:
String and the icon should probably be passed by reference, as they are in the other methods, to avoid copying & copy constructors.